notebooks/name-entity-extraction.ipynb

```diff
@@ -0,0 +1,785 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    multilingual_examples = ["A Budapest, il y a une grande piscine que les touristes visitent.",
</pre><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/graphcore/optimum-graphcore-fork/pull/17/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>

Reply:
<p>Ty, I forgot to actually put that in translate instead of using my brain to practice for fun.</p>